### PR TITLE
Nodejs base structure

### PR DESCRIPTION
**Pull request by SreenathES**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const adminRoutes = require('./modules/admin/admin.routes');
+const studentRoutes = require('./modules/student/student.routes');
+
+const app = express();
+
+app.use(express.json());
+app.use('/instructor', instructorRoutes);
+app.use('/admin', adminRoutes);
+app.use('/student', studentRoutes);
+app.listen(80);
```
Comment (mishelshaji) [inline server/app.js:13]: Add a blank line above.

File: server/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "server",
```
Comment (mishelshaji) [inline server/package.json]: Change the app name.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: server/app.js (hunk 1 above)
Comment (SreenathES) [reply server/app.js:13]: added blank line

File: server/package.json (hunk 2 above)
Comment (SreenathES) [reply server/package.json]: App name changed

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**